src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffendersResourceCourtTransfersTest.java

```diff
@@ -24,7 +24,7 @@
 public class OffendersResourceCourtTransfersTest extends ResourceTest {
```

Comment:
we possibly din't need this test, it almost certainly covered by the newer tests  but keep until it can be assessed